src/databricks/labs/remorph/reconcile/query_config.py

```diff
@@ -0,0 +1,71 @@
+from databricks.labs.remorph.reconcile.constants import SourceType
+from databricks.labs.remorph.reconcile.recon_config import (
+    ColumnMapping,
+    Schema,
+    Table,
+    Transformation,
+)
+
+
+class QueryConfig:  # pylint: disable=too-many-instance-attributes)
```

Comment:
```suggestion
class QueryConfig:
```
You are not allowed to disable linter warnings.

Convert instance attribute usage to methods.

Reply:
fixed.